print/ghostscript/Portfile

```diff
@@ -162,6 +180,17 @@ variant cups description {Enable CUPS driver} {
     configure.args-replace  --disable-cups --enable-cups
 }
 
+variant ghostpdl description {Install GhostPDL} {
+    configure.args-delete --without-pcl --without-xps --without-gpdl
```

Comment:
Is the opposite of each of these a corresponding `--enable-*` statement? If so, then use a configure.args-replace statement for each of them. For example:

`configure.args-replace --without-pcl --enable-pcl
`

Reply:
The inverse would be `--with-pcl=gpcl6`, I'll change it to use that.